src/app/issuer/components/issuer-create/issuer-create.component.html

```diff
@@ -1,12 +1,11 @@
-<form-message></form-message>
-<div class="topbar">
-	<div class="l-containerxaxis">
-		<div class="topbar-x-heading">{{ 'Issuer.createInstitution' | translate }}</div>
-		<div class="topbar-x-subheading">{{ 'Issuer.awardBadeges' | translate }}</div>
+<div class="oeb page-padding">
+	<form-message></form-message>
+
+	<div class="oeb-headline-container-sm">
+		<h1 hlmH1 class="tw-text-purple">{{ 'Issuer.createInstitution' | translate }}</h1>
+		<p hlmP class="tw-font-normal">{{ 'Issuer.awardBadeges' | translate }}</p>
```

Comment:
To make it consistent with the other pages use a double headline:
```
<h1 hlmH1 class="tw-font-normal tw-text-purple">
     TEXT
</h1>
<h1 hlmH1 class="tw-font-black tw-text-purple">
     TEXT
</h1>
```

Reply:
updated.